tests/ci.rs

```diff
@@ -42,6 +42,7 @@ use routing::{Client, Data, Event, FullId, Node, PlainData, RequestContent, Requ
               ResponseMessage};
 
 const GROUP_SIZE: usize = kademlia_routing_table::GROUP_SIZE as usize;
+const QUORUM_SIZE: usize = 5;
```

Comment:
I think it was deliberately left private so we're forced to use [`dynamic_quorum_size`](https://github.com/maidsafe/kademlia_routing_table/blob/master/src/lib.rs#L331-L337).

However, we could remove the local `GROUP_SIZE` variable once https://github.com/maidsafe/kademlia_routing_table/pull/19 is merged since I think it's only there to cast the existing `u8` from routing table.